network/README.md

```diff
@@ -30,6 +30,26 @@ This library is designed to be integrated in your program.
   h.ServeUnix("root", "test_network")
 ```
 
+### Example using Windows named pipes:
+
+```go
+import "github.com/docker/go-plugins-helpers/network"
+import "github.com/docker/go-plugins-helpers/sdk"
+
+d := MyNetworkDriver{}
+h := network.NewHandler(d)
+
+config := sdk.WindowsPipeConfig{
+  // open, read, write permissions for everyone 
+  // (uses Windows Security Descriptor Definition Language)
+  SecurityDescriptor: "S:(ML;;NW;;;LW)D:(A;;0x12019f;;;WD)",
```

Comment:
I'm not a windows guy, but this looks insanely complex.
To make it "easy" for non-Windows savvy to provide plugin support for windows would it make sense to include some of these as constants in `windows_listener.go`